packages/actions/index.ts

```diff
@@ -718,12 +718,17 @@ export const logoutAccount = () => {
 export const removeAccount = () => {
   return async (dispatch, getState) => {
     const { address, privateKeyBuffer } = getUser(getState())()
-    await mnemonicStorage.remove()
-    await hashedPasswordStorage.remove()
-    await creditProtocol.deleteChannelID(address, privateKeyBuffer)
-    const payload = { hasStoredUser: false, user: undefined }
-    dispatch(displaySuccess(accountManagement.logoutSuccess))
-    dispatch(setState(payload))
+
+    try {
+      await creditProtocol.deleteChannelID(address, privateKeyBuffer)
+      await mnemonicStorage.remove()
+      await hashedPasswordStorage.remove()
+      const payload = { hasStoredUser: false, user: undefined }
+      dispatch(displaySuccess(accountManagement.logoutSuccess))
+      dispatch(setState(payload))
+    } catch (e) {
+      dispatch(displaySuccess(errorTitle))
```

Comment:
Minor nits/curiosities:

 1. The error here, `e` isn't being used. There could be a helpful message in there, and burying errors has a tendency to make it more difficult to troubleshoot/diagnose issues if/when something unexpected were to go awry in the try block (e.g: if `Storage#remove()` doesn't work on a device for some reason). Displaying the error message to the user, or at least logging it, would mitigate that concern.

 2. I think `errorTitle` might be a typo. Above, `genericError` is being destructured out of `language`, and I think it may be a typo, too. In the language files themselves, `logoutError` is being added.

 3. The `displaySuccess()` function is being used here to display an error, which is confusing. Might want to introduce a `displayError()` function, even if it is just an alias, to clarify the code here.